x-pack/plugins/upgrade_assistant/public/app.tsx

```diff
@@ -14,7 +14,7 @@ import { UpgradeAssistantTabs } from './components/tabs';
 
 export const RootComponentUI: React.StatelessComponent = () => (
   <Fragment>
```

Comment:
Can we change this to a `div className="upgradeAssistantRoot"` and set that class to have `display: flex`? Then we could put the `data-test-subj` on this root element.